packages/entity-browser/src/util/api/entities.js

```diff
@@ -17,8 +18,7 @@ export const defaultModelTransformer = json => {
   return model
 }
 
-export function fetchModel(entityName, transformer = defaultModelTransformer) {
-  return request(`entities/${entityName}/model`)
-    .then(resp => resp.body)
-    .then(json => transformer(json))
+export function* fetchModel(entityName, transformer = defaultModelTransformer) {
+  const resp = yield call(requestSaga, `entities/${entityName}/model`)
```

Comment:
since GET is the default (as used here), i think we dont need "getRequestSaga" and can remove it?